src/App.jsx

```diff
@@ -1,28 +1,18 @@
 // React Imports
-import React, { useEffect, useMemo, useState } from 'react';
+import React from 'react';
 // Inrupt Imports
```

Comment:
Could remove comments for unused imports Line 3 to 5

Reply:
updated